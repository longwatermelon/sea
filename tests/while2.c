fn main() -> int {
    let x:int=5;
    let ans:int=0;
    while (x!=7) {
        x=x+1;
        ans=ans+1;
    }
    return ans;
}
