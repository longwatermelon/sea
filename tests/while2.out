RETURN_CODE:2
STDOUT:
